code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -22,15 +22,16 @@
 	fitted = FEMALE_UNIFORM_TOP
 
 /obj/item/clothing/under/rank/engineering/atmospheric_technician
-	desc = "It's a jumpsuit worn by atmospheric technicians."
+	desc = "It's a jumpsuit worn by atmospheric technicians. It has minor protection from fire."
 	name = "atmospheric technician's jumpsuit"
 	icon_state = "atmos"
 	item_state = "atmos_suit"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 60, "acid" = 20)

```

Comment:
why not just move this line (or, well, the line it's copying) to /obj/item/clothing/under/rank/engineering?

Reply:
other jumpsuits didnt do that so i didnt want to do that